examples/counter-vanilla/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Redux basic example</title>
+    <script src="https://npmcdn.com/redux@latest/dist/redux.min.js"></script>
```

Comment:
You can also use brcdn.org for any packages that don't include umd builds in npm :)